app/parse-siti-payment-file/validate.js

```diff
@@ -1,11 +1,17 @@
-const checkInvoiceTotal = require('./invoice-totals')
+const getInvoiceTotal = require('./invoice-totals')
+
+const validateLineTotals = (invoiceHeaders) => {
+  return invoiceHeaders
+    .map(a => a.totalValue === getInvoiceTotal(a.lines, 'value'))
+    .every(b => b === true)
```

Comment:
You can do this in one go: `.every(a => a.totalValue === getInvoiceTotal(a.lines, 'value'))`